_posts/0000-01-02-brendanrhyshewer.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
There are 22 letters in the English alphabet...No that isn't right there are 26...oh I guess i forgot u r a b...me too thanks...me too thanks bzzz

Reply:
yis